src/index.html

```diff
@@ -6,7 +6,41 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <main class="main">
```

Comment:
Do U really need one more container here?

Reply:
It's optional, but I decided to use a <div> wrapper to avoid specifying additional styles for the <body>